app/javascript/mastodon/features/account/components/header.js

```diff
@@ -323,6 +314,8 @@ class Header extends ImmutablePureComponent {
                 </div>
               )}
 
+              {account.get('id') !== me && <AccountNoteContainer account={account} />}
```

Comment:
Wait, why did you remove ability to set note for yourself?

Reply:
Because we don't load relationship for the user itself the note is never loaded, so while it was possible to set it, it would always show up as empty.

Reply:
Is it possible to fix it somehow? Because I find note for myself useful, just like chatting with myself in messengers… you can leave some things for later. I actively used that in Discord.